queries/highlights.scm

```diff
@@ -191,21 +198,44 @@
   (keyword_sort)
   (keyword_compute)
   (keyword_comment)
-  (keyword_partitioned)
   (keyword_location)
   (keyword_cached)
   (keyword_uncached)
   (keyword_lines)
   (keyword_stored)
-  (keyword_location)
   (keyword_partitioned)
-  (keyword_cached)
   (keyword_analyze)
   (keyword_rewrite)
-  (keyword_statistics)
```

Comment:
I think `statistics` and `stats` should be `@type.qualifier` and not `@keyword`

Reply:
I removed these from here because they were duplicated in `@type.qualifier` already 👍